internal/generate/clusterserviceversion/clusterserviceversion_updaters_test.go

```diff
@@ -332,6 +332,49 @@ var _ = Describe("findMatchingDeploymentAndServiceForWebhook", func() {
 			Expect(service.GetName()).To(Equal(serviceName1))
 		})
 	})
+
+	Context("match webhookconfigs with same service and path", func() {
```

Comment:
It's worth putting function-specific tests in their own context
```suggestion
	Context("crdGroups", func() {
		It("matches webhookconfigs with same service and path", func() {
```